superset-frontend/src/dashboard/components/nativeFilters/CascadePopover.tsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useCallback } from 'react';
+import { ExtraFormData, styled, t } from '@superset-ui/core';
+import Popover from 'src/common/components/Popover';
+import Icon from 'src/components/Icon';
+import { Pill } from 'src/dashboard/components/FiltersBadge/Styles';
+import { CascadeFilterControl, FilterControl } from './FilterBar';
+import { Filter, CascadeFilter } from './types';
+
+interface CascadePopoverProps {
+  filter: CascadeFilter;
+  visible: boolean;
+  onVisibleChange: (visible: boolean) => void;
+  onExtraFormDataChange: (filter: Filter, extraFormData: ExtraFormData) => void;
+}
+
+const StyledTitleBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.grayscale.light4};
+  margin: -5px -16px; // to override default antd padding
```

Comment:
```suggestion
  margin: ${({ theme }) => theme.gridUnit * -1}px; ${({ theme }) => theme.gridUnit * -4}px; // to override default antd padding
```
Hopefully -4px is close enough to -5px?